reference/sectionD_references.inc

```diff
@@ -0,0 +1,17 @@
+
+.. _sectionD:
+
+Extensible Markup Language (XML) 1.1 (Second Edition) http://www.w3.org/TR/2006/REC-xml11-20060816 (16 August 2006)
+
+Mathematical Markup Language (MathML) Version 2.0 (Second Edition) http://www.w3.org/TR/2003/REC-MathML2-20031021/ (21 October 2003)
+
+Namespaces in XML 1.1 (Second Edition) http://www.w3.org/TR/2006/REC-xml-names11-20060816/ (16 August 2006)
+
+RFC 2119: Key words for use in RFCs to Indicate Requirement Levels http://www.ietf.org/rfc/rfc2119.txt (March 1997)
+
+Unicode 1.1.0  https://www.unicode.org/charts/ (June 1993)
+
+XML Information Set (Second Edition) http://www.w3.org/TR/2004/REC-xml-infoset-20040204/ (4 February 2004)
+
+XML Linking Language (XLink) Version 1.0 http://www.w3.org/TR/2001/REC-xlink-20010627/ (27 June 2001)
```

Comment:
Prefer https://www.w3.org/TR/2001/REC-xlink-20010627/.

Reply:
What should we do about this one? https://www.w3.org/TR/xlink/ refers to XLink 1.1...

Reply:
hmmmm....looks like we could switch to XLink 1.1 and everything should be fine...still uses the same xlink namespace.

Or we just stick to referencing the XLink 1.0 spec via that version specific URL and play it safe :)